packages/ember-toucan-core/src/components/form/fields/input.hbs

```diff
@@ -45,10 +44,37 @@
         @value={{@value}}
         @onChange={{@onChange}}
         @hasError={{this.hasError}}
+        {{on "input" this.handleCount}}
         ...attributes
       />
     </field.Control>
-    {{#if @error}}
+    {{#if (has-block "secondary")}}
+      <div
+        class="type-xs-tight text-critical mt-1.5 flex justify-between
+          {{if this.hasMoreThanOneError 'items-start' 'items-center'}}"
+      >
+        {{#if @error}}
+          <field.Error
+            @isInline={{true}}
+            class="flex-3"
+            id={{field.errorId}}
+            @error={{@error}}
+            data-error
+          />
+        {{/if}}
+
+        <div class="text-body-and-labels mt-1.5 flex-1 text-right">
```

Comment:
We'll also want to remove `mt-1.5` here as it's adding additional margin top as margin is built-in to `<Error`